core/tee/tee_svc_cryp.c

```diff
@@ -1206,6 +1199,36 @@ TEE_Result tee_obj_attr_copy_from(struct tee_obj *o, const struct tee_obj *src)
 	return TEE_SUCCESS;
 }
 
+static bool is_gp_legacy_des_key_size(TEE_ObjectType type, size_t sz)
+{
+	return IS_ENABLED(CFG_COMPAT_GP10_DES) &&
+	       ((type == TEE_TYPE_DES && sz == 56) ||
+		(type == TEE_TYPE_DES3 && (sz == 112 || sz == 168)));
```

Comment:
(commit "core: include parity in DES/DES3 key sizes")
```patch
-		(type == TEE_TYPE_DES3 && (sz == 112 || sz == 168)));
+		(type == TEE_TYPE_DES3 && (sz == 56 || sz == 112 || sz == 168)));

Reply:
DES3 with 56 bit key isn't mentioned in the spec, I don't think it's valid.

Reply:
Yes you're right. Discard my comment.